sunpy/util/decorators.py

```diff
@@ -96,7 +96,7 @@ def deprecated_func(*args, **kwargs):
 
             category = SunpyDeprecationWarning
 
-            warnings.warn(message, category, stacklevel=2)
+            warnings.warn(message, category)
```

Comment:
```suggestion
            warnings.warn(message, category, stacklevel=2)
```

This will completely break the decorator, see the description of [stacklevel in the warning docs](https://docs.python.org/3/library/warnings.html#warnings.warn)